src/burnchains/burnchain.rs

```diff
@@ -365,6 +365,35 @@ impl Burnchain {
         })
     }
 
+    /// the expected sunset burn is:
+    ///   total_commit * (progress through sunset phase) / (sunset phase duration)
+    pub fn expected_sunset_burn(&self, burn_height: u64, total_commit: u64) -> u64 {
+        if burn_height < self.pox_constants.sunset_start
```

Comment:
Is `.sunset_start` in units of reward cycles or block heights?  Because, I see it get used for both here.